src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carCost = 40;
+
+  if (days >= 7) {
+    return carCost * days - 50;
```

Comment:
Don't repeat yourself: you've repeated the `cardCost * days` logic 3 times.

Also check point 1 in checklist. For now it's not clear what is `40`, `50` and `20`